### DISCUSSION TITLE
Comment shown twice after posting

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#109346] [Mobile]
GitHub Mobile beta for Android 
https://github.com/community/community/discussions/108148#discussioncomment-8530634
![Screenshot_20240220-091542.png](https://github.com/community/community/assets/2119212/0d7e6c96-3590-4a06-8dc2-90fe7aa60302)

**Comment by iskae** [comment]
Hey @jsoref 👋 

Unfortunately, I was not able to reproduce the issue. Also checked your original comment on other thread, but it was related to emoji states. Am I missing something? Can you elaborate more on the issue you are experiencing?

**Reply by jsoref** [reply]
I think it happens often when I post something that it (temporarily) appears twice. It goes away with a refresh, so it feels like it's a glitch in how the app interacts with the server when it posts a message.

It's possible it gets two notifications or two similar notifications and renders both? 🤷
